### DISCUSSION TITLE
Candy Fridge and Hoover Induction Hob integration

### DISCUSSION THREAD

**Discussion by ojoaosemtil** [ofalvai/home-assistant-candy#112] [Device support improvements]
Hello,

Is it possible to integrate Candy Fridge and Hoover Induction Hob?
Hoover and Candy work both in Candy Simply-fi app and Hoover Wizard app

**Comment by ofalvai** [comment]
If those apps can talk to the device, then yes, it's possible. I won't be able to do it on my own, please see this guide about how you can help: https://github.com/ofalvai/home-assistant-candy#my-device-isnt-supported-can-you-help
